code/modules/research/designs/comp_board_designs.dm

```diff
@@ -3,6 +3,7 @@
 /datum/design/board
 	name = "Computer Design ( NULL ENTRY )"
 	desc = "I promise this doesn't give you syndicate goodies!"
+	build_type = IMPRINTER
 	materials = list(/datum/material/glass = 1000)
```

Comment:
uhh

Reply:
Just tested it, as you can see it does nto have a build_path, meaning it shouldn't be able to be printed. The test also confirmed this.